modules/monitoring-configuring-alertmanager-to-send-notifications-for-user-defined-alert-routing.adoc

```diff
@@ -37,15 +37,15 @@ route:
   - name: Default
   routes:
   - matchers:
-    - "service = prometheus-example-monitor" <1>
-    receiver: <receiver> <2>
+    - "service = prometheus-example-monitor" #<1>
+    receiver: <receiver> #<2>
 receivers:
 - name: Default
 - name: <receiver>
 #  <receiver_configuration>
 ----
-<1> Specifies which alerts match the route. This example shows all alerts that have the `service="prometheus-example-monitor"` label.
-<2> Specifies the receiver to use for the alerts group.
+<1> Specify the target alerts. This example targets all alerts that have the `service="prometheus-example-monitor"` label.
```

Comment:
"target alerts" is a bit unclear for me (but maybe my bias as a long-time Alertmanager user).

Reply:
No, I agree with you, I think it would be a lot clearer to change the wording here and in the previous procedure as well. I will change it to: "Specify labels to match your alerts."